ack/readers/awin_advertiser/reader.py

```diff
@@ -65,6 +69,10 @@ def request(self):
             build_url, params=payload
         )
         json_response = response.json()
+        # if report is not a campaign report, remove tags from the response
+        if self.report_type != REPORT_TYPES[2]:
+            for element in json_response:
+                element.pop('tags')
```

Comment:
Why do you need to remove tags? Just to understand here

Reply:
They are not necessary for the use case I am using the AWIN reader for, however if you prefer I keep them on the reader and could create a copy of ACK with this modification instead done outside of the repo. Please let me know.

Reply:
Yes it is better to answer specific needs outside of the official repo. If other people want to use your reader they'll not have the same needs as you so keep it general.
Another solution is to add an boolean argument to your reader like `--remove-tag` for example and to only do your modification if this argument is set to True.

Reply:
have made this an optional argument now

Reply:
thanks